pkg/nugetconfig/parse_testcase.go

```diff
@@ -0,0 +1,13 @@
+package nugetconfig
+
+import "github.com/aquasecurity/go-dep-parser/pkg/types"
+
+var (
+	NuGetPackagesConfig = []types.Library{
+		{"Microsoft.AspNet.WebApi", "5.2.2"},
+		{"Microsoft.AspNet.WebApi.Client", "5.2.2"},
+		{"Microsoft.AspNet.WebApi.Core", "5.2.2"},
+		{"Microsoft.AspNet.WebApi.WebHost", "5.2.2"},
+		{"Newtonsoft.Json", "6.0.4"},
+	}
```

Comment:
Could you write down how to generate `packages.config`? You can refer to other parsers.